src/gfx_layout_fallback.cpp

```diff
@@ -118,23 +118,22 @@ FallbackParagraphLayout::FallbackVisualRun::FallbackVisualRun(Font *font, const
 	this->glyph_to_char.reserve(char_count);
 
 	/* Positions contains the location of the begin of each of the glyphs, and the end of the last one. */
```

Comment:
I guess this comment needs to updating.

Reply:
Hmm, I don't think it's even necessary, it was there to explain the +1.